src/js_realm.hpp

```diff
@@ -212,6 +213,7 @@ struct RealmClass : ClassDefinition<T, SharedRealm> {
         {"create", wrap<Realm::create>},
         {"delete", wrap<Realm::delete_one>},
         {"deleteAll", wrap<Realm::delete_all>},
+        {"isValid", wrap<Realm::is_object_valid>},
```

Comment:
All the other methods which operate on objects are not static methods so perhaps we should keep this as is for consistency. I also wouldn't be opposed to also validating that the passed in object is from the given realm.